adl/rot13adl/rot13node.go

```diff
@@ -0,0 +1,197 @@
+/*
+	rot13adl is a demo ADL -- its purpose is to show what an ADL and its public interface can look like.
+	It implements a "rot13" string: when creating data through the ADL, the user gives it a regular string;
+	the ADL will create aninternal representation of it which has the characters altered in a reversable way.
+
+	It provides reference and example materal, but it's very unlikely you want to use it in real situations ;)
+
+	There are several ways to move data in and out of the ADL:
+
+		- treat it like a regular IPLD map:
```

Comment:
This probably isn't as interesting as using the substrate directly but incorporating this into the example might be nice, to show the higher-level API that will mostly be consumed by users. I suppose the example could set a string property, then `Subtrate()` and inspect the property underneath it?

Reply:
I see the tests exercising all of this, I think it'd be worth raising some more of that functionality up into the example since it's doing the initial work of demonstrating how this all works and you need to set up the mental model properly up front. IMO.

Reply:
Example added.  Thanks for the kick.